src/main/webapp/WEB-INF/views/car/allCars.jsp

```diff
@@ -0,0 +1,64 @@
+<%@ taglib prefix="c" uri ="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+  <title>All cars</title>
+</head>
+<style>
+  table {
+    border: 1px solid;
+    text-align: center;
+    margin-top: 20%;
+    width: 100%;
+  }
+  tr, td {
+    border: 1px solid;
+  }
+  #innerTable {
+    width: 100%;
+    height: 100%;
+    margin: 0 0 0 0;
```

Comment:
so interesting

Reply:
maybe width and height redundant, because it is working without this properties